MacOS 10.15.7 disk.Partitions() is failing

**Describe the bug**
When compiling on MacOS 10.15.7, specifically the [Github Actions virtual environment for MacOS](https://github.com/actions/virtual-environments/releases/tag/macos-10.15%2F20201003.1) released yesterday, gopsutil is no longer compiling:
```
# github.com/shirou/gopsutil/disk
Error: ../../../go/pkg/mod/github.com/shirou/gopsutil@v2.20.8+incompatible/disk/disk_darwin.go:67:51: cannot use stat.Mntfromname[:] (type []byte) as type []int8 in argument to common.IntToString
Error: ../../../go/pkg/mod/github.com/shirou/gopsutil@v2.20.8+incompatible/disk/disk_darwin.go:68:49: cannot use stat.Mntonname[:] (type []byte) as type []int8 in argument to common.IntToString
Error: ../../../go/pkg/mod/github.com/shirou/gopsutil@v2.20.8+incompatible/disk/disk_darwin.go:69:50: cannot use stat.Fstypename[:] (type []byte) as type []int8 in argument to common.IntToString
Error: ../../../go/pkg/mod/github.com/shirou/gopsutil@v2.20.8+incompatible/disk/disk_darwin.go:85:43: cannot use stat.Fstypename[:] (type []byte) as type []int8 in argument to common.IntToString
```

The code this references is https://github.com/shirou/gopsutil/blame/master/disk/disk_darwin.go#L67 and was last touched 6 years ago.

**Environment (please complete the following information):**
 - [ ] Windows: [paste the result of `ver`]
 - [ ] Linux: [paste contents of `/etc/os-release` and the result of `uname -a`]
 - [x ] Mac OS: System Version: macOS 10.15.7 (19H2)
 - [ ] FreeBSD: [paste the result of `freebsd-version -k -r -u` and  `uname -a`]
 - [ ] OpenBSD: [paste the result of `uname -a`]

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -49,7 +49,7 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:e7b553682850788c6b266faaf25218ce901558e541102d12b2998cad99f6b4fb"
+  digest = "1:0afe79d034c63eea8c5977401f5fc135394751c1213ee6c991b0c6bec44f8f60"
   name = "golang.org/x/sys"
   packages = [
     "internal/unsafeheader",
@@ -59,7 +59,7 @@
     "windows/svc/mgr",
   ]
   pruneopts = "UT"
-  revision = "76b94024e4b621e672466e8db3d7f084e7ddcad2"
+  revision = "e3ed0017c21142d3345308d92dcc7ad7020040c6"
 
 [[projects]]
   branch = "v3"

--- a/disk/disk_darwin.go
+++ b/disk/disk_darwin.go
@@ -64,9 +64,9 @@ func PartitionsWithContext(ctx context.Context, all bool) ([]PartitionStat, erro
 			opts += ",nodev"
 		}
 		d := PartitionStat{
-			Device:     common.IntToString(stat.Mntfromname[:]),
-			Mountpoint: common.IntToString(stat.Mntonname[:]),
-			Fstype:     common.IntToString(stat.Fstypename[:]),
+			Device:     common.ByteToString(stat.Mntfromname[:]),
+			Mountpoint: common.ByteToString(stat.Mntonname[:]),
+			Fstype:     common.ByteToString(stat.Fstypename[:]),
 			Opts:       opts,
 		}
 		if all == false {
@@ -82,5 +82,5 @@ func PartitionsWithContext(ctx context.Context, all bool) ([]PartitionStat, erro
 }
 
 func getFsType(stat unix.Statfs_t) string {
-	return common.IntToString(stat.Fstypename[:])
+	return common.ByteToString(stat.Fstypename[:])
 }

